.npmignore

```diff
@@ -0,0 +1,13 @@
+.eslintignore
```

Comment:
Do we need an NPM ignore? I'm under the impression we won't be publishing this project to NPM?

Reply:
I am not sure. I can remove it and we can add it back later if we find it is needed.